src/components/Navigation.js

```diff
@@ -0,0 +1,25 @@
+import '../css/nav.css';
+
+const Navigation = () => {
+    return  <nav class="nav-container display-flex primary-clr p-rl-6">
+        <a href="./index.html" class="nav-link font-size-2pt6">
```

Comment:
dont use anchor tag(<a>) as it will result in a non-SPA application which is not a desired outcome we want when using react and react-router-dom use Link component from react-router-dom instead

Reply:
Yes, i will do that after adding all of the components at one go.